dss/storage/checkout.py

```diff
@@ -131,3 +132,25 @@ def touch_test_file(dst_bucket: str, replica: Replica) -> bool:
         return True
     except Exception as e:
         return False
+
+def status_file_name(execution_id: str) -> str:
+    return f"checkout/status/{execution_id}.json"
+
+def put_status(status: str, execution_id: str, default_checkout_bucket: str=None,
+               dst_replica: Replica=None, dst_bucket: str=None, dst_location: str=None):
+    handle = Config.get_blobstore_handle(Replica.aws)
```

Comment:
I did not see much value in keeping the status of checkouts in a replica specific location. This would also require user to pass replica in order to check the status of the checkout.